setup.py

```diff
@@ -27,7 +27,7 @@
         ],
     },
     tests_require=[
-        'mock',
+        'mock==1.0.1',
```

Comment:
I tried leaving it alone but the build would fail as it requires a higher version of setuptools. I can force the install of a newer setuptool/pip but then @lra might not like it. Idk. @lra does it really matter in this case?